packages/experimental/src/components/AboutModal/_about-modal.scss

```diff
@@ -0,0 +1,185 @@
+//
+// Copyright IBM Corp. 2020, 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../global/styles/carbon-settings';
+@import '../../global/styles/project-settings';
+@import '@carbon/themes/scss/themes';
+@import 'carbon-components/scss/globals/scss/_vars.scss';
+@import 'carbon-components/scss/components/link/link';
+@import 'carbon-components/scss/components/modal/modal';
+@import 'carbon-components/scss/components/tabs/tabs';
+
+.#{$exp-prefix}-about-modal.#{$exp-prefix}-about-modal-scroll-enabled
+  .bx--modal-container {
+  height: 100%;
+  padding-bottom: $spacing-09;
+}
+
+.#{$exp-prefix}-about-modal.#{$exp-prefix}-about-modal-scroll-enabled
+  .bx--modal-container
+  .#{$exp-prefix}-modal-content {
+  display: grid;
+  overflow-y: auto;
+}
+
+.#{$exp-prefix}-modal-content
+  .bx--modal-content.#{$exp-prefix}-about-modal-content {
+  padding-bottom: $spacing-07;
+}
+
+.#{$exp-prefix}-about-modal .#{$exp-prefix}-modal-content .bx--modal-footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+}
+
+.#{$exp-prefix}-about-modal.#{$exp-prefix}-about-modal-light-theme {
+  @include carbon--theme($carbon--theme--white, true);
+}
+
+.#{$exp-prefix}-about-modal.#{$exp-prefix}-about-modal-light-theme
+  .bx--modal-footer {
+  @include carbon--theme($carbon--theme--g100, true);
+}
+
+.#{$exp-prefix}-about-modal.#{$exp-prefix}-about-modal-light-theme
+  .#{$exp-prefix}-about-modal-tab-container {
+  @include carbon--theme($carbon--theme--white, true);
+}
+
+.#{$exp-prefix}-about-modal.#{$exp-prefix}-about-modal-dark-theme {
+  @include carbon--theme($carbon--theme--g100, true);
+}
+
+.#{$exp-prefix}-about-modal.#{$exp-prefix}-about-modal-dark-theme
+  .bx--modal-footer {
+  @include carbon--theme($carbon--theme--white, true);
+}
+
+.#{$exp-prefix}-about-modal.#{$exp-prefix}-about-modal-dark-theme
+  .#{$exp-prefix}-about-modal-tab-container {
+  @include carbon--theme($carbon--theme--g100, true);
+}
+
+.#{$exp-prefix}-about-modal .bx--modal-footer {
+  position: relative;
+  flex-direction: column;
+  grid-row: auto;
+  justify-content: center;
+  padding-left: $spacing-05;
+  background-color: $ui-background;
+}
+
+.#{$exp-prefix}-about-modal {
+  background-color: $ui-background;
+  .bx--modal-content.#{$exp-prefix}-about-modal-content {
+    @include carbon--type-style('body-short-02');
+
+    padding-top: $spacing-02;
+    overflow-y: scroll;
```

Comment:
```suggestion
    overflow-x: hidden;
    overflow-y: auto;
```

Reply:
Probably only need this once after losing the enable scroll property.

Reply:
Because of the way `ModalBody` is build from carbon, I needed to add this part as well for it to scroll properly. I have added the `overflow-x: hidden` part as suggested.